detection_rules/devtools.py

```diff
@@ -103,7 +103,11 @@ def build_release(config_file, update_version_lock: bool, generate_navigator: bo
     package = Package.from_config(config=config, verbose=verbose)
 
     if update_version_lock:
-        loaded_version_lock.manage_versions(package.rules, save_changes=True, verbose=verbose)
+        if not RULES_CONFIG.bypass_version_lock:
+            loaded_version_lock.manage_versions(package.rules, save_changes=True, verbose=verbose)
+        else:
+            click.echo('WARNING: Skipping version lock update'
+                       'Unset `bypass_version_lock` in the config to enable version lock updates')
```

Comment:
```suggestion
        if RULES_CONFIG.bypass_version_lock:
            click.echo('WARNING: Skipping version lock update'
                       'Unset `bypass_version_lock` in the config to enable version lock updates')
        else:
            loaded_version_lock.manage_versions(package.rules, save_changes=True, verbose=verbose)
```

Nit should this be reversed to not have what might sound like a double negative (not bypass)?

Reply:
I think this was removed.